apps/rush-lib/src/rush-schema.json

```diff
@@ -103,6 +103,20 @@
         "additionalProperties": false,
         "required": [ "packageName", "projectFolder" ]
       }
+    },
+    "rushHooks": {
+      "description": "Hooks allow to a customized script to run at a specified time.",
+      "type": "object",
+      "properties": {
+        "postCommand": {
+          "description": "The list of scripts to run after every Rush command.",
```

Comment:
>"The list of scripts to run after every Rush command." [](start = 25, length = 54)

Which shell do these commands run under?  i.e. how do we make them behave consistently on different operating systems?

Might be useful to look at how the NPM tool invokes its scripts.